charts/dbildungscloud-iam-keycloak/templates/deployment.yaml

```diff
@@ -39,6 +42,28 @@ spec:
                   key:  admin-password
             - name: KC_PROXY
               value: "edge"
+            - name: DB_BASE_URL
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.secrets.name }}
+                  key: db.url.jdbc
+            - name: DB_NAME
+              valueFrom:
+                secretKeyRef:
```

Comment:
db name should not be created from the secrets, the variable is differentiated per instance and is overwritten by us in the pipeline after the database creation and during the keycloak installation (similar to the variable for the ingres-url)